lib/fluent/plugin/out_datadog.rb

```diff
@@ -127,14 +127,13 @@ def write(chunk)
 
   def send_to_datadog(data)
     @my_mutex.synchronize do
-      retries = 0
```

Comment:
I don't think we want to start from 1 here as we don't retry in the first attempt, WDYT ? I'd rather leave `retries = 0` here.

Reply:
I can see that. The log message says "attempts", not "retries". Should we change the log message too? Or I suppose we could change the name of the variable to "attempts" and continue to initialize at 0. I don't feel strongly either way, just as long as the log message makes sense.

Reply:
I am fine with renaming the variable to `attemps`, initializing it to `0` and incrementing it before logging the the warning message, WDYT ?

Reply:
I would just remove the first `retries` in the first hand and keep the code the same for now, i.e just remove `#L137`
ll investigate the issue you opened https://github.com/DataDog/fluent-plugin-datadog/issues/5 and do a bit of refactoring. How does it sound to you ?

Reply:
Sounds fine to me. Do you want me to update this PR

Reply:
Yes please

Reply:
Yes please